doc/lsst.meas.algorithms/creating-a-reference-catalog.rst

```diff
@@ -128,14 +128,54 @@ Monitor the log file in a new terminal with:
 
     tail -f convert-gaia.log
 
-Check the log ouput after several hours: ``ConvertReferenceCatalogTask`` reports progress in 1% intervals.
+Check the log output after several hours: ``ConvertReferenceCatalogTask`` reports progress in 1% intervals.
 
 .. _lsst.meas.algorithms-refcat-ingest:
 
 4. Ingest the files into the butler
 ===================================
 
-When ``convertReferenceCatalog`` has finished, it will print the two commands you need to run to register the new refcat dataset type and ingest your converted output into it.
+When ``convertReferenceCatalog`` has finished, a new directory (named ``gaia-refcat/`` in the example above) will now exist containing all reference catalogs in the LSST format.
+
+.. note::
```

Comment:
Looking at the generated text, the `note` breaks up the flow quite a lot, and is irrelevant for anyone following the docs with a fresh refcat. I wonder if this would be better as a `section 5: what to do with existing refcats`?

Reply:
Good idea. I'll move it as you suggest.